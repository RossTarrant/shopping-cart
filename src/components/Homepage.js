export default function Homepage(){
    return(
        <div className="grid grid-cols-1 md:grid-cols-2">
            <img className="lg:rotate-45 mt-36 lg:ml-12" alt="fancy-shoes" src="https://www.stylist.co.uk/images/app/uploads/2020/04/01165256/gucci-trainers.png?w=1200&h=1&fit=max&auto=format%2Ccompress"/>
            <div className="text-center text-5xl lg:text-7xl xl:text-9xl text-slate-800 font-bold m-12">
                <h1 className="">Shoes.</h1>
                <h1 className="">Shoes.</h1>
                <h1 className="">Shoes.</h1>
                <h1 className="">Shoes.</h1>
            </div>
        </div>
    )
}