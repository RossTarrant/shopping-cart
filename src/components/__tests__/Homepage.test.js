import React from "react";
import { render, screen } from "@testing-library/react";
import Homepage from "../Homepage";

describe("Homepage component", () => {

    render(<Homepage />);

    it("should render four headings", () => {
      const headings = screen.getAllByText("Shoes.")
      expect(headings.length).toBe(4);
    });

    it("renders homepage", () => {
        const { container } = render(<Homepage />);
        expect(container).toMatchSnapshot();
    });

});