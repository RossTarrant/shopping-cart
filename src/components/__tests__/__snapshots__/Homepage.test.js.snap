// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component renders homepage 1`] = `
<div>
  <div
    class="grid grid-cols-1 md:grid-cols-2"
  >
    <img
      alt="fancy-shoes"
      class="lg:rotate-45 mt-36 lg:ml-12"
      src="https://www.stylist.co.uk/images/app/uploads/2020/04/01165256/gucci-trainers.png?w=1200&h=1&fit=max&auto=format%2Ccompress"
    />
    <div
      class="text-center text-5xl lg:text-7xl xl:text-9xl text-slate-800 font-bold m-12"
    >
      <h1
        class=""
      >
        Shoes.
      </h1>
      <h1
        class=""
      >
        Shoes.
      </h1>
      <h1
        class=""
      >
        Shoes.
      </h1>
      <h1
        class=""
      >
        Shoes.
      </h1>
    </div>
  </div>
</div>
`;
